src/System.Drawing.Primitives/tests/SizeFTests.cs

```diff
@@ -77,6 +77,57 @@ public void ArithmeticTest(float width, float height)
         }
 
         [Theory]
+        [InlineData(1000.234, 2400.933)]
```

Comment:
Shouldn't all these values be float, rather than double (i.e. `1000.234f` instead of `1000.234`)?